hono-router.hbs

```diff
@@ -149,7 +149,7 @@ function returnHandler(
     case 'Number':
     case 'String':
       return ctx.text(
-        data.toString() as string,
+        data?.toString() as string,
```

Comment:
I'm guessing that the issue is that typescript thinks that data is optional.. As we know that it's defined in this case, would it be better with `data!.toString()`?

Reply:
On line 146 it says it's optional `data?: string | object,`

Reply:
Could we just do a if(!data) { return } before the switch statement?

Reply:
Sure thing. TypeScript is also happy with that

![image](https://github.com/markusahlstrand/tsoa-hono/assets/8496063/2b203068-2cc9-4b75-8848-c2fda36a7f82)

Reply:
Like it.. Feels cleaner as the data should be defined in the switch statement